firecloud_project_test.py

```diff
@@ -97,6 +97,26 @@ def test_secure_network(self):
     self.assertEqual([x['name'] for x in firewall['properties']['rules']],
                      ['allow-icmp', 'allow-internal', 'leonardo-ssl'])
 
+  def test_private_ip_google_access(self):
+    """Verifying changes are made with the privateIpGoogleAccess option."""
+    self.context.properties['highSecurityNetwork'] = True
+    self.context.properties['enableFlowLogs'] = True
+    self.context.properties['privateIpGoogleAccess'] = True
+    resources = firecloud_project.generate_config(self.context)['resources']
+
+    # Network has enabled custom static route
+    # Not sure why, but the route does not appear in the config but gets created in the deployment
```

Comment:
This may be because we use an action to create the route. In the deployment, this is the output: 

```
private-google-access-route
gcp-types/compute-v1:compute.routes.insert

private-google-access-route

action
gcp-types/compute-v1:compute.routes.insert

destRange
199.36.153.4/30

name
private-google-access-route

network
$(ref.network.selfLink)

nextHopGateway
projects/$(ref.project.projectId)/global/gateways/default-internet-gateway

project
$(ref.project.projectId)
```